bot.py

```diff
@@ -1,14 +1,15 @@
-import json
 import os
 from datetime import datetime
 from dotenv import load_dotenv
 import discord
 from discord import app_commands
+from modules.utils import *
```

Comment:
Avoid "import *" as is clutters up the namespace and you don't clearly know what you've imported, which hinders readability.
```suggestion
from modules.utils import get_map_url, get_file_data, write_file_data
```
If the import become too long, it might be interesting to wrap up some methods or further separate the files.